guests/templates/checklist/guest_merch_deadline.html

```diff
@@ -9,7 +9,8 @@
   </div>
   <script type="text/javascript">
       var showOrHideTaxInfo = function() {
-          setVisible('#tax-info', $.val('selling_merch') === {{ c.OWN_TABLE }})
+          setVisible('#tax-info', $.val('selling_merch') === {{ c.OWN_TABLE }});
+          $.field('tax_phone').prop('required', $.val('selling_merch') === {{ c.OWN_TABLE }});
```

Comment:
This fixes the bug that was preventing anyone from submitting the merch deadline as 'not selling.'